src/client/java/dev/technici4n/moderndynamics/client/model/MdModels.java

```diff
@@ -40,7 +39,8 @@ public class MdModels {
     };
 
     public static ModelState preRotated(BlockModelRotation rotation, float preAngle) {
-        Transformation preRotation = new Transformation(null, new Quaternion(new Vector3f(0, 0, 1), preAngle, true), null, null);
+
+        Transformation preRotation = new Transformation(null, new Quaternionf(0, 0, 1, 0), null, null);
```

Comment:
Sorry I'm clueless about geometry so I have no idea what the line previously there or what I replaced it with does, but it works somehow.